app/src/main/java/org/fossasia/openevent/general/auth/EditProfileViewModel.kt

```diff
@@ -25,14 +26,22 @@ class EditProfileViewModel(
     private val mutableMessage = SingleLiveEvent<String>()
     val message: LiveData<String> = mutableMessage
 
+    var avatarUpdated = MutableLiveData<Boolean>(false)
```

Comment:
Variable `avatarUpdated` is not used in this view model.

Reply:
I used it in fragment and it should be normal variable not livedata, I'll update it